### PR TITLE
Slack api

### PR DESCRIPTION
**Pull request by dvargas92495**
Creating a table for employee inbox messages. Also an API endpoint to start receiving events from Slack and adding them to our DB.

The agent event loop will always be checking its inbox for unread messages during its agent loop in order to know what work it needs to do

### CODE REVIEW

File: data/schema.ts [hunk 1]
```diff
@@ -485,3 +485,15 @@ export const employeesHistory = mysqlTable("employees_history", {
     .notNull()
     .default(""),
 });
+
+export const employeeInboxMessages = mysqlTable("employee_inbox_messages", {
```
Comment (dvargas92495) [inline data/schema.ts:489]: I expect this to evolve drastically over time:
- note no `employeeUuid` column here. we likely should - I think in Slack's case this translates to every employee being its own "Slack App" which is kind of wild to think about. To short cut this, we're just gonna have the SamePage app receive everything for now. The abstraction to think about here that's useful is imagining SamePage itself being an agent that is sort of a messages representative for all actual employees.
- no `text` column - all raw text we're storing in S3. In the future, they should all ideally be in our AtJson representation if that proves to be useful.
- using `context` as a catchall to save all event data that could be useful

File: api/apps/slack/events/post.ts
Comment (dvargas92495) [inline api/apps/slack/events/post.ts:1]: going to start centralizing all api stuff from the separate repos back into the mono repo

**Comment by dvargas92495** [comment]
Table deployed

File: api/apps/slack/events/post.ts
Comment (mdroidian) [reply api/apps/slack/events/post.ts:1]: does this mean something like this https://github.com/samepage-network/github-samepage/blob/main/api/oauth/post.ts will end up in the mono repo?

**Review by mdroidian — COMMENTED** [review]
🎉

**Pull request merged**